storage/factory/common.go

```diff
@@ -117,3 +134,77 @@ func getLastEpochFromDirNames(epochDirs []string) (uint32, error) {
 
 	return epochsInDirName[0], nil
 }
+
+func getLastEpochFromStorage(
```

Comment:
this function does 3 things: computes a file path, instantiate a storer (cacher + persister) and search some data inside.

Reply:
extracted a function